lib/scout_apm/app_server_load.rb

```diff
@@ -51,7 +51,7 @@ def data
     ensure
       # Sometimes :database_engine and :database_adapter can cause a reference to an AR connection.
       # Make sure we release all AR connections held by this thread.
-      ActiveRecord::Base.connection_handler.clear_active_connections! if Utils::KlassHelper.defined?("ActiveRecord::Base")
+      ActiveRecord::Base.connection_handler.clear_active_connections!(:all) if Utils::KlassHelper.defined?("ActiveRecord::Base")
```

Comment:
Hello - I wanted to open the same PR (when running tests this deprecation warning gets quite verbose). In any case I think this is a safer way to implement this (backwards compatibility):

```ruby
      if ActiveRecord::Base.connection_handler.respond_to?(:clear_active_connections!)
        ActiveRecord::Base.connection_handler.clear_active_connections!(:all)
      elsif ActiveRecord::Base.respond_to?(:clear_active_connections!)
        # ~> Rails 3.2 -> https://apidock.com/rails/ActiveRecord/Base/clear_active_connections%21/class
        ActiveRecord::Base.clear_active_connections!
      end
```

Reply:
Ah you mean that `clear_active_connections!` doesn't take any arguments in Rails 3? That could be actually. But I wonder why no test is failing. I am gonna look into it.

Reply:
Oh lol, this code isn't even tested?!